tools/depends/target/Makefile

```diff
@@ -97,6 +97,7 @@ pythonmodule-pil: $(ZLIB) libjpeg-turbo libpng freetype2 python26
 libsdl2: $(LINUX_SYSTEM_LIBS)
 libxslt: libgcrypt
 ffmpeg: $(ICONV) $(ZLIB) bzip2 libvorbis $(FFMPEG_DEPENDS)
+libcec: platform
```

Comment:
No i also need to rebuild them if anything in the addons buildsys changes ;). I think the problem with non packaged addons has nothing to do with this at all. I somehow missread this changeset i think.

ATM if something in projects/cmake changes i prune tools/depeneds/target/binary-addons and this triggers a rebuild. A change in tools/depends/target/platform/Makefile basically does the same because it git cleans all (including binary addons). So this should be covered already.